docker/centreon-broker/api_control.py

```
@@ -0,0 +1,42 @@
+    )
+
+@app.post("/restart")
+def restart_cbd():
+    global cbd_proc
+    # Stop only the process we started
+    if cbd_proc is not None:
+        cbd_proc.terminate()  # sends SIGTERM
+        try:
+            cbd_proc.wait(timeout=5)
+        except Exception:
+            pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)